core/field_label_serializable.js

```diff
@@ -36,14 +36,16 @@ goog.require('Blockly.utils');
  * Class for a non-editable, serializable text field.
  * @param {*} opt_value The initial value of the field. Should cast to a
  *    string. Defaults to an empty string if null or undefined.
- * @param {string=} opt_class Optional CSS class for the field's text.
+ * @param {Object=} opt_config A map of options used to configure the field.
+ *    See the documentation for a list of properties this parameter supports.
+ *    https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/label-serializable#creation
```

Comment:
If you use {@link http....} it will add a link when this is used to generate documentation.

Reply:
Fixed. I will fix for all other fields separately.